### DISCUSSION TITLE
讨论独立博客的自主性与平台性推荐的矛盾

### DISCUSSION THREAD

**Discussion by zhanglong12138** [timqian/chinese-independent-blogs#1254] [General]
readme中提到将个人博客通过一定的方法推荐给其他人，这涉及到不同博客内容的统一整合，而这可能与博客的独立性矛盾。
比如不同博客内容储存格式的不同，如果是主流工具搭建的博客，或许可以在作者允许的情况下被爬取整理。另外，这种平台性质的推荐，会让头部博客突出出来，被更多人看到，最终反而会让酒香的巷子“越来越深”。
最令我担忧的是，这种脱颖而出的头部博客可能深谙营销之道，采用在其他主流平台上的做法发表内容，按套路出牌。说到底，找到一个个人博客，浏览他的文章内容，汲取他的思想精华，这是一个主动寻找内容的过程，这本身其实与被推荐的浏览存在根本性的矛盾，以上是我的一些观点，拙见。

**Comment by timqian** [comment]
是的，没有完美的推荐方式，有些博主会受益更多，有些收益小一些。但无论如何，是有收益的一件好事
